CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ All notable changes to this project will be documented in this file.
 
 - ci: Remove Nexus steps from build, mirror and release workflows ([#1056]).
   Also remove the old release workflow.
+- spark: remove 3.5.2 and update dependencies for 3.5.5 ([#1094])
```

Comment:
I think _update dependencies for 3.5.5_ can go under the Changes heading.

Reply:
https://github.com/stackabletech/docker-images/pull/1094/commits/ebbca87814735747ba4cf367b94092704b31836a